core/src/main/kotlin/utils/Extensions.kt

```diff
@@ -76,6 +76,11 @@ fun ApplicationCall.pagingOptions(defaultSortProperty: SortProperty): PagingOpti
     return PagingOptions(limit, offset, sortProperties)
 }
 
+/**
+ * Convert the [String] path of a URL to a [Long] entity ID. If the conversion fails, throw a [UrlPathFormatException].
+ */
+fun String.checkedToLong() = toLongOrNull() ?: throw UrlPathFormatException("Invalid entity ID: '$this'.")
```

Comment:
Maybe this should also throw the exception if the results is `< 0`.

Reply:
To my knowledge also `0` is an invalid ID, because the first one is always `1`.
Added a check for `< 1`, and because this function became now more specific, renamed it to `toLongId()`.